sopel/plugins/handlers.py

```diff
@@ -128,47 +130,47 @@ def is_loaded(self):
         raise NotImplementedError
 
     def setup(self, bot):
-        """Setup the plugin with the ``bot``
+        """Setup the plugin with the ``bot``.
```

Comment:
```suggestion
        """Run the plugin's setup routine.
```

Technically, this method doesn't actually do anything related to the ``bot`` anyway; it's the plugin itself that needs to run its setup routine.